kratos/python_interface/__init__.py

```diff
@@ -12,6 +12,27 @@
 # import core library (Kratos.so)
 from Kratos import *
 
+def __ModuleInitDetail():
+    """
+    Configure the parallel environment.
+    This is done before initializing the kernel to ensure that MPI
+    and the parallel DataCommunicator are initialized when the Kernel is built.
+    It is defined as a function to avoid polluting the Kratos namespace with local variables.
+    """
+    if "--using-mpi" in sys.argv[1:]:
+        try:
+            import KratosMultiphysics.mpi
+        except ModuleNotFoundError:
+            msg = [
+                "\nRequesting MPI support through the \"--using-mpi\" command line option, ",
+                "\nbut the mpi environment could not be initialized. The most likely cause ",
+                "\nfor this warning is that this Kratos installation was compiled without ",
+                "\nMPI support."
+            ]
+            Logger.PrintWarning("KRATOS INITIALIZATION WARNING:", "".join(msg))
```

Comment:
will this already be directed through the logger that prints only on one rank?
Or is this not yet initialized?